### DISCUSSION TITLE
Don't store the Image block as base64 and add support for all file adapters

### DISCUSSION THREAD

**Discussion by dimaip** [keystonejs/keystone#5314] [Feature requests]
It's a bit unfortunate that the default `fields-content` Image block tries to base64 encode the images.
That leads to `PayloadTooLargeError: request entity too large` when trying to save. And even thought it's possible to increase the upload limit, it would be a very strange solution performance-wise trying to store all images inside a database...

Currently there exists a Cloudinary image block, but I couldn't find any blocks for S3 or the file adapter.
I think it would have been a much nicer API if the default image block just supported different adapters, with a config like this:

```
fields: {
    body: {
      type: Content,
      blocks: [[Content.blocks.image, { adapter }]],
    },
  },
```

**Comment by gautamsi** [comment]
@mitchellhamilton is this something we can expect in new interface? image adapter for uploading?

**Comment by bladey** [comment]
Thank you for the suggestion @dimaip.

**Comment by vigneshakshan** [comment]
could you tell me how to increase the upload limit?
@dimaip
